### DISCUSSION TITLE
I have an HDD, not an SSD. Will Elementary work on my computer?

### DISCUSSION THREAD

**Discussion by peanutbutterbar** [elementary/installer#744] [Q&A] [answered]
Hi, relative beginner here who's interested in installing Elementary. Having looked at the recommended specifications for Elementary, I'd like to ask if my laptop can handle the OS. I have an HDD, not an SSD. Is that OK? Will Elementary work on my computer?

**Comment by ghost** [comment, marked as answer]
Yes, Elementary will work on HDD. SSD is much faster than HDD, so elementary will work a little bit slower.

**Reply by peanutbutterbar** [reply]
Thank you so much!
